src/ilmm.jl

```diff
@@ -1,19 +1,16 @@
 """
-    ILMM(fs, H)
+    ILMM(f, H)
+
 An Instantaneous Linear Mixing Model (ILMM) -- a distribution over vector-
 valued functions. Let `p` be the number of observed outputs, and `m` the number of latent
 processes, then `H`, also known as the mixing matrix, is a `p x m` matrix whose column
-space spans the output space. The latent processes are represented a Gaussian process `f`.
+space spans the output space. `H` "mixes" a latent multi-output AbstractGP `f`.
```

Comment:
Is "mixes" maybe a bit too ambiguous or vague? Or given we have explained what `H` is are we just introducing the term "mixes"

Reply:
I was referencing the fact that call it the mixing matrix above, but I agree we should make it more precise.

How about something like
 ```suggestion
space spans the output space. Given a sample from `f`, the value of the ILMM at any input `x` for all outputs is
```julia
H * [f((x, 1)), f((x, 2)), ..., f((x, m))]
```
```

Reply:
That makes it super clear! Once that is commited I am happy with this being merged!